pkg/lifecycle/lifecycle.go

```diff
@@ -22,7 +22,7 @@ type Asset struct {
 // Dependencies holds the necessary filesystem,
 // assets versions map, version rules and methods to apply the lifecycle rules in the target branch
 type Dependencies struct {
-	rootFs            billy.Filesystem
+	RootFs            billy.Filesystem
```

Comment:
Is this field accessed outside of this package?

Reply:
Yes it is used by auto package